interfaces/builtin/opengl.go

```diff
@@ -35,6 +35,12 @@ const openglConnectedPlugAppArmor = `
   /var/lib/snapd/lib/gl/ r,
   /var/lib/snapd/lib/gl/** rm,
 
+  # Solus LDM locations
+  /var/lib/snapd/hostfs/usr/lib*/glx-provider/** rm,
+  /var/lib/snapd/hostfs/usr/lib*/libcuda* rm,
+  /var/lib/snapd/hostfs/usr/lib*/libn* rm,
+  /var/lib/snapd/hostfs/usr/lib*/lib{GL,EGL}* rm,
```

Comment:
Well the comment in the context does explain that, its required for Solus NVIDIA libraries, which are accessed under /usr/lib{,64}/glx-provider due to the /usr/lib*/libGL type symlinks, and the rest of the libs (actual NVIDIA ones, not conflicting OpenGL ones) live in the main library directory. And the non-Solus part was never present because the rules were created for Arch Linux which doesn't actually have full apparmor confinement, so it was never exposed until now.